cloud/libvirt/actuators/machine/utils/domain.go

```diff
@@ -10,13 +11,16 @@ import (
 	"strings"
 	"time"
 
+	"k8s.io/client-go/kubernetes"
```

Comment:
Anything out of the scope of the libvirt api won't belong here and will need to be refactored as part of https://github.com/openshift/cluster-api-provider-libvirt/pull/38